ui/raidboss/data/06-ew/raid/p7s.ts

```diff
@@ -10,6 +10,7 @@ import { TriggerSet } from '../../../../../types/trigger';
 export interface Data extends RaidbossData {
   purgationDebuffs: { [role: string]: { [name: string]: number } };
   purgationDebuffCount: number;
+  rootsCounter?: boolean;
```

Comment:
This is a little bit hard for me to follow.  It's named "counter" but you're checking against both `false` and `undefined` and `true` separately?

Reply:
It's just a way to count to three with a boolean (1 byte) rather than int, but it is less intuitive. Alternatively it could be renamed and just flip odd/even with the boolean.

Reply:
I changed the name to be more understandable as secondRoots. Still utilizing the undefined for the first roots.